src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Morphisms/Constructors.jl

```diff
@@ -201,3 +201,31 @@ function base_change(phi::Any, IP::AbsProjectiveScheme)
   return result, psi
 end
 
+########################################################################
+# Rational maps
+########################################################################
+@doc raw"""
+    rational_map(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map; check::Bool=true )
+
+Given a homomorphism ``f : T → S`` of the `homogeneous_coordinate_ring`s of `Q` and `P`, respectively, 
+construct the associated rational map of projective varieties.
```

Comment:
This is defined for schemes but the documentation is for varieties.  
We have a type for projective varieties. I would suggest to use that?

Reply:
I thought about using the `AbsProjectiveVariety`, but in practice this will kill the user. We want to be able to compute with schemes which nevertheless have the properties of varieties. These are checked, eventually based on the actual type of the input.